fabric/src/main/java/com/wynntils/fabric/WynntilsModFabric.java

```diff
@@ -14,12 +14,14 @@ public class WynntilsModFabric implements ClientModInitializer {
     @Override
     public void onInitializeClient() {
         Optional<ModContainer> wynntilsMod = FabricLoader.getInstance().getModContainer("wynntils");
-        if (wynntilsMod.isEmpty()) {
```

Comment:
Why is this better? The old code follows our traditional style of:

if (thingsAreNotOk) return;

doRealStuff;